### PR TITLE
[auth] Make UI consistent with new Batch design

### PR DESCRIPTION
**Pull request by daniel-goldstein**
This aligns the auth styling with the new batch UI styling. Nothing has really changed about the structure of the pages. This one should be considerably easier than the batch overhaul, just three pages `/user`, `/users` and `/roles` (I think roles were added to auth but never fleshed out, we never use this page. But that's for another PR..).

I manually checked that I didn't break any of the Batch pages after moving the utils file into web_common, but a sanity check on that would be great.

### CODE REVIEW

File: auth/auth/templates/users.html [hunk 1]
```diff
@@ -1,53 +1,76 @@
...
+        <div><input type="checkbox" name="is_service_account" value="1" /> Service Account</div>
+        <input type="hidden" name="_csrf" value="{{ csrf_token }}" />
+        <div class='flex justify-end'>
+          {{ submit_button('Create') }}
+        </div>
+      </div>
+    </form>
+  </div>
+  <div>
+    <h1 class='text-2xl font-light mb-4'>Delete User</h1>
```
Comment (iris-garden) [inline auth/auth/templates/users.html:28]: these two forms are a little close together, would you mind adding a left margin to the delete user one?

File: auth/auth/templates/users.html [hunk 2]
```diff
@@ -1,53 +1,76 @@
...
+  </thead>
+  <tbody class='border border-collapse border-slate-50'>
+    {% for user in users %}
+    <tr class='bg-slate-50 font-light'>
+      <td class='text-center'>{{ user['id'] }}</td>
+      <td>{{ user['username'] }}</td>
+      <td>{{ user['login_id'] }}</td>
+      <td>{{ user['hail_identity'] }}</td>
+      <td>{{ user['state'] }}</td>
+      <td class='text-center'>{{ user['is_developer'] }}</td>
```
Comment (iris-garden) [inline auth/auth/templates/users.html:70]: could these be check marks and x's instead of 1s and 0s?

File: auth/auth/templates/users.html [hunk 3]
```diff
@@ -1,53 +1,76 @@
...
-        <td>{{ user['state'] }}</td>
-        <td class="numeric-cell">{{ user['is_developer'] }}</td>
-        <td class="numeric-cell">{{ user['is_service_account'] }}</td>
-      </tr>
-      {% endfor %}
-    </tbody>
-  </table>
+<br />
+<h1 class='text-2xl font-light mb-4'>Users</h1>
+<table class="table-auto">
```
Comment (iris-garden) [inline auth/auth/templates/users.html:50]: i'd like this table to match the one on the batches page if possible, spanning the full page, with the white background and lines between rows, and with the headers aligned the same way as the cell text (all left-aligned except center for both the header and text in the first row)

**Review by iris-garden — CHANGES_REQUESTED** [review]
looks good, just a couple notes!

File: auth/auth/templates/users.html (hunk 1 above)
Comment (daniel-goldstein) [reply auth/auth/templates/users.html:28]: Added some spacing between the two elements in the flexbox

File: auth/auth/templates/users.html (hunk 3 above)
Comment (daniel-goldstein) [reply auth/auth/templates/users.html:50]: Done! I centered the check marks and Xs but let me know if you'd rather them be left aligned.

**Comment by daniel-goldstein** [comment]
@iris-garden bump

**Pull request merged**